app/metrics.yaml

```diff
@@ -154,6 +154,16 @@ events:
     notification_emails:
       - fenix-core@mozilla.com
     expires: "2020-03-01"
+  search_fragment_cache_access:
```

Comment:
Could you fill out the (https://github.com/mozilla/data-review/blob/master/request.md)[data request form] for this? :)

Reply:
@boek [already did](https://github.com/mozilla-mobile/fenix/pull/6197#issuecomment-547030238)!

Reply:
I'm a dummy, don't know how I missed it!